suggest/elasticsearch/src/main/java/com/spotify/heroic/suggest/elasticsearch/SuggestBackendKV.java

```diff
@@ -21,17 +21,11 @@
 
```

Comment:
my formatter kept messing these lines up... my intelij is set to use the google style guide

@hexedpackets which one do we use in heroic?

https://github.com/spotify/heroic/pull/624/files?w=1 shows this file without all the whitespace changes

Reply:
The style XML for intellij is in the repo - https://github.com/spotify/heroic/blob/master/idea/code-style.xml

Reply:
weird - i think the reformatting must have just been really off in this file -- reformatting with the heroic style didn't change many things back.